.github/workflows/publish.yml

```diff
@@ -38,8 +38,6 @@ jobs:
         # uses: pypa/gh-action-pypi-publish@27b31702a0e7fc50959f5ad993c78deac1bdfc29
         uses: pypa/gh-action-pypi-publish@v1.4.2
         with:
-          # PyPI user
-          user: ${{ secrets.PYPI_USER }}
           # Password for your PyPI user or an access token
           password: ${{ secrets.PYPI_PASS }}
```

Comment:
this needs to be the token pass, is this  a token pass? better to get it right

as per doc, https://github.com/pypa/gh-action-pypi-publish, we should name the secret 

```
    password: ${{ secrets.PYPI_API_TOKEN }}
```

Reply:
@pindge the naming is inconsistent across repos, in several places it's `secrets.PYPI_TOKEN`. I'm not sure if it makes a difference, though deciding on a common convention would be nice